nebula-components/components/_flyout.scss

```diff
@@ -21,7 +21,8 @@
     left: 0;
 
     &::after {
-      left: $nb-flyout-triangle-positioning;
+      bottom: -($nb-spacing-unit / 10 * 3.5);
```

Comment:
Still have a magic number here.  Has this been tested to work across screen sizes?  I feel just a pixel unit might work better that is assigned to a descriptive local variable.
I'm guessing this value has something to do with how far the flyout is placed from it's parent and the size of the triangle, so it's sensible to tie those two together so that this value is auto generated.  As is changing either of the former values I mentioned would cause this UI pattern to break down.